src/Components/JoinExistingList.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
+import { useHistory } from 'react-router-dom';
+
+export default function JoinExistingList({ setToken }) {
+  const [input, setInput] = useState('');
+  let history = useHistory();
+
+  const handleClick = (event) => {
```

Comment:
This isn't a blocker, but something you could consider here would be to talk as a team about how you'd like to handle error messages so that the user has the same experience throughout the app. Establishing what you want now will make for easier refactoring than going through things later! 

For example, the Join List form sends alerts in the click handler here, but the [Add Item form](https://github.com/the-collab-lab/tcl-12-smart-shopping-list/blob/main/src/Components/AddItem.jsx) handles them in the form itself.